ui/desktop/app/templates/application.hbs

```diff
@@ -21,30 +21,35 @@
 
       <header.utilities as |utilities|>
         {{#if this.showWindowActions}}
-          <Hds::Button
-            @icon='x'
-            @size='small'
-            @text={{t 'actions.close'}}
-            @color='critical'
-            class='button-window-close'
-            {{on 'click' this.close}}
-          />
-          <Hds::Button
-            @icon='x'
-            @size='small'
-            @color='secondary'
-            @text={{t 'actions.fullscreen'}}
-            class='button-window-fullscreen'
-            {{on 'click' this.toggleFullScreen}}
-          />
-          <Hds::Button
-            @icon='x'
-            @size='small'
-            @color='secondary'
-            @text={{t 'actions.minimize'}}
-            class='button-window-minimize'
-            {{on 'click' this.minimize}}
-          />
+          <Hds::ButtonSet>
+            <Hds::Button
+              @icon='monitor'
```

Comment:
Note: The closest icon in the [library](https://helios.hashicorp.design/icons/library) I found was `monitor`, to represent a user triggering a full screen.

Reply:
We are pausing on ui/desktop changes. Please disregard.